src/main/java/io/r2dbc/postgresql/util/Assert.java

```diff
@@ -19,12 +19,16 @@
 import reactor.util.annotation.Nullable;
 
 import java.io.File;
+import java.net.MalformedURLException;
+import java.net.URL;
 
 /**
  * Assertion library for the implementation.
  */
 public final class Assert {
 
+    private static final String CLASSPATH_PREFIX = "classpath:";
```

Comment:
That isn't a generally supported URL scheme. It only works because Spring is on the class path.